### DISCUSSION TITLE
CAP_NET_RAW Privilege

### DISCUSSION THREAD

**Discussion by lisbethseldon** [projectdiscovery/naabu#773] [General]
When I try to use scripts I get this message and it just finishes.

[INF] Current naabu version 2.1.6 (latest)
[INF] Running host discovery scan
[INF] Running SYN scan with CAP_NET_RAW privileges

I installed library, go, naabu again and again. But, I still get the same message. Please help me!!!

**Comment by brenocss** [comment]
> [INF] Running SYN scan with CAP_NET_RAW privileges

this message is intended


>  and it just finishes

can you give a command example to reproduce?

**Comment by awetz** [comment]
[](url)> > [INF] Running SYN scan with CAP_NET_RAW privileges
> 
> this message is intended
> 
> > and it just finishes
> 
> can you give a command example to reproduce?

i have same issue on linux vps but working on oracle vbox.

https://ibb.co/pPcWXLP

**Comment by mrvcoder** [comment]
hello , I have the same problem on linux vps server but working on my local machine

**Comment by syphon1c** [comment]
I did experience this same issue but this was related to running naabu on a Linux instance on Docker on my M2 Macbook Pro, installing Nmap also failed but with a little more details I could use to resolve the issue. 

Enable the option `Use Rosetta for x86/amd64 emulation on Apple Silicon`

Option path:
Docker Desktop > Settings > Features in Development > Use Rosetta for x86/amd64 emulation on Apple Silicon

**Comment by Mzack9999** [comment]
This is only an informative message that notifies the user about the kind of scan which is ongoing:
- If the user doesn't have enough privileges, a connect scan is performed
- if the binary has the `CAP_NET_RAW` capability (ref: https://linux.die.net/man/7/capabilities - in short the process has the ability to manipulate raw packets) then a SYN scan is performed (only one packet is used instead of a full TCP handshake).

More detailed info about the scan difference can be found at the following links:
- Connect scan: https://nmap.org/book/scan-methods-connect-scan.html
- SYN scan: https://nmap.org/book/synscan.html

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
